src/engine/Input/Pointer.ts

```diff
@@ -80,6 +86,44 @@ export class PointerEvent extends GameEvent<any> {
 };
 
 /**
+ * Wheel Events
+ *
+ * Represents a mouse wheel event. See [[Pointers]] for more information on
+ * handling point input.
+ */
+export class WheelEvent extends GameEvent<any> {
```

Comment:
Upon further thought, I think this will not work in an way which is obvious.

According to spec, the result is essentially a 3D Vector. It describes a physical user action to a direction and magnitude in a 3D space.  

Since Excaliburjs does not have an internal 3D Vector model I don't think there is a reason to implement this. Giving the delta of X, Y, and Z is easy enough for any developer. Doing all the checks to figure out the direction seems a little overkill when most people will just a simple check of deltaY.